template/src/serviceWorker.ts

```diff
@@ -19,26 +19,23 @@ const isLocalhost = Boolean(
     )
 );
 
-export default function register() {
+export default function register(config: any) {
```

Comment:
The `config` parameter deserves a bit more strict type declaration, doesn' it?
Currently, it's just an object with optional `onUpdate` and `onSuccess` parameters, which both receive a service worker registration object if available.